lib/takeaway.rb

```diff
@@ -0,0 +1,82 @@
+require_relative 'twilio_sender'
+
+class Takeaway
+
+  attr_reader :menu, :twilio_sender
+  attr_accessor :selection
+
+  def initialize twilio_sender
+    @menu = {
+      pizza: 9.99,
+      chicken: 2.99,
+      kebab: 4.99,
+      beer: 0.99,
+      cider: 0.99,
+      wine: 7.99
+      }
+    @selection = {}
+    @twilio_sender = twilio_sender
+  end
+
+  def display_menu
+    @menu.each do |item, price|
+      "#{item.to_s.capitalize} : £#{price}"
+    end
+  end
+
+  def select_items item, quantity
+    if item_has_already_been_chosen? item
+      update_existing_quantity quantity, item
+    else
+      @selection[item] = quantity
+    end
+  end
+
+  def total
+    total_price = 0.0
+    selection.each { |item, quantity| get_price item
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.